packages/widget-message-meet/src/components/spark-logo/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from './styles.css';
+
+import logo from './logo-spark.png';
+
+// eslint-disable-next-line func-style,arrow-body-style
+const SparkLogo = () => {
+  return (
+    <div className={classNames(`spark-logo`, styles.logo)}>
+      <img role="presentation" src={logo} />
```

Comment:
Good call. We should still be able to make this a component though right? Just have the proper css values?